pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -257,5 +258,13 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: memoryThrottlingFactor %v must be greater than 0 and less than or equal to 1.0", *kc.MemoryThrottlingFactor))
 	}
 
+	if !path.IsAbs(kc.PodLogsPath) {
```

Comment:
should this use path or filepath? what is our expectation of this path on windows? will it break joining or handling to get a path in the form "C:\path\to\logs"?

Reply:
Oh, I didn't know there are two implementations in Go library :)
Yup, we need platform dependent one here.
Fixed.